kiwi/cli.py

```diff
@@ -206,6 +207,8 @@ def get_global_args(self):
                         'vmx type is now a subset of oem, --type set to oem'
                     )
                     value = 'oem'
+                if arg == '--shared-cache-dir' and not value:
+                    value = os.sep + Defaults.get_shared_cache_location()
```

Comment:
I wonder why ```os.sep + ...``` is needed here ? Do you had a particular reason why defaults.py can't simply provide ```/var/cache/kiwi``` with the leading "/" ? Actually the former default setting from docopt did that too

Reply:
The former default in cli returned `/var/cache/kiwi` and in `get_shared_cache_location` the leading '/' was [stripped off](https://github.com/OSInside/kiwi/blob/5a0d13f233eafce325c307c5cc4a3cbec64268ce/kiwi/defaults.py#L185), returning `var/cache/kiwi`

Given the comment, I thought to preserve that behavior would be a good idea unless the reasons for stripping off the lead '/' are no longer valid or have changed.